routers/repo/branch.go

```diff
@@ -213,11 +221,68 @@ func loadBranches(ctx *context.Context) []*Branch {
 			ctx.ServerError("GetLatestPullRequestByHeadInfo", err)
 			return nil
 		}
+		mergeMovedOn := false
 		if pr != nil {
 			if err := pr.LoadIssue(); err != nil {
 				ctx.ServerError("pr.LoadIssue", err)
 				return nil
 			}
+			if repo, ok := repoIDToRepo[pr.BaseRepoID]; ok {
+				pr.HeadRepo = repo
```

Comment:
```suggestion
				pr.BaseRepo = repo
```
🤔

Reply:
I get a nil pointer exeption - if i add guillep2k suggestion it works